nbextensions/appCell2/widgets/appCellWidget.js

```diff
@@ -1041,16 +1030,6 @@ define(
                 }
             }
 
-            // LIFECYCYLE API
-
-            function doEditNotebookMetadata() {
-                Narrative.editNotebookMetadata();
-            }
-
-            function doEditCellMetadata() {
-                Narrative.editCellMetadata(cell);
-            }
```

Comment:
replaced each of these functions (which are only used once) with the direct call to the Narrative function